src/main/resources/public/raml/resourceTypes/rules.raml

```diff
@@ -1,4 +1,13 @@
-description: This class is used to test rules on Eiffel events. Testrules must be enabled in Eiffel Intelligence.
+description: This class is used for anything rules related. It's currently used to test rules on eiffel events
+             and to fetch the active rules file content. Test rules must be enabled in Eiffel Intelligence
```

Comment:
Should we be more specific here? Maybe spell out the property name? Or perhaps link to GitHub documentation about configuration.